internal/vulnstore/postgres/getupdateoperationdiff.go

```diff
@@ -0,0 +1,147 @@
+package postgres
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/jackc/pgx/v4"
+	"github.com/jackc/pgx/v4/pgxpool"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/libvuln/driver"
+)
+
+func getUpdateDiff(ctx context.Context, pool *pgxpool.Pool, prev, cur uuid.UUID) (*driver.UpdateDiff, error) {
+	// Query takes two update IDs and returns rows that only exist in first
+	// argument's set of vulnerabilities.
+	const query = `WITH
```

Comment:
don't we want this defined at the package level so it is not constructed on every function call?

Reply:
It's not constructed on every call. The placement of constants only affects their scope.